packages/file_selector/file_selector_platform_interface/lib/src/method_channel/method_channel_file_selector.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+
+import 'package:file_selector_platform_interface/file_selector_platform_interface.dart';
+import 'package:meta/meta.dart';
+
+const MethodChannel _channel = MethodChannel('plugins.flutter.io/file_picker');
```

Comment:
Make the channel name match the plugin name: `plugins.flutter.io/file_selector`

Reply:
Oops haha